client/hack/README.md

```diff
@@ -0,0 +1,81 @@
+# Scripts User Guide
+
+This README documents:
+* What update-crd.sh and update-generated-code.sh do
+* When and how to use them
+
+## update-generated-code.sh
+
+This is the script to update clientset/informers/listers and API deepcopy code using [code-generator](https://github.com/kubernetes/code-generator).
+
+Make sure to run this script after making changes to /client/apis/volumesnapshot/v1beta1/types.go.
+
+Run: ./hack/update-generated-code.sh from the project root directory.
+
+Once you run the script, you will get an output as follows:
+```bash
+Generating deepcopy funcs
+Generating clientset for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/clientset
+Generating listers for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/listers
+Generating informers for volumesnapshot:v1beta1 at github.com/kubernetes-csi/external-snapshotter/client/informers
+
+```
+
+
+## update-crd.sh
+
+This is the script to update CRD yaml files under /client/config/crd/ based on types.go file.
+
+Make sure to run this script after making changes to /client/apis/volumesnapshot/v1beta1/types.go.
+
+Follow these steps to update the CRD:
+
+* Run ../hack/update-crd.sh from client directory, new yaml files should have been created under ./config/crd/
```

Comment:
This path should also be changed:
../hack/update-crd.sh

Reply:
Please go thru the README and make sure the directory structure referenced in the doc is correct after moving everything under client.